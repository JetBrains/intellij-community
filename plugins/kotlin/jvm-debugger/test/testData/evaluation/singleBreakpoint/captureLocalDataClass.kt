fun main() {
    data class DC(val a: Int)

    //Breakpoint!
    val x = 1
}

// EXPRESSION: DC(42).a
// RESULT: 42: I