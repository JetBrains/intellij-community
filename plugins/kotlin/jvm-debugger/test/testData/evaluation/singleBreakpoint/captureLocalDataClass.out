LineBreakpoint created at captureLocalDataClass.kt:5
Run Java
Connected to the target VM
captureLocalDataClass.kt:5
Compile bytecode for DC(42).a
Disconnected from the target VM

Process finished with exit code 0
